src/config/api.ts

```diff
@@ -1,5 +1,5 @@
 export const API_CONFIG = {
-  BASE_URL: import.meta.env.VITE_API_BASE_URL || "http://localhost:3000/api",
+  BASE_URL: import.meta.env.VITE_API_BASE_URL || "https://finfx-backend-dev.onrender.com",
```

Comment:
```suggestion
  BASE_URL: import.meta.env.VITE_API_BASE_URL || "https://finfx-backend-dev.onrender.com/dev",
```

Actually we need to add /dev as the suffix in order for it to work.